src/main/kotlin/it/pagopa/wallet/scheduler/exceptions/SemNotAcquiredException.kt

```diff
@@ -0,0 +1,4 @@
+package it.pagopa.wallet.scheduler.exceptions
+
+class SemNotAcquiredException(semName: String, throwable: Throwable? = null) :
```

Comment:
Those new exception could extend a common exception (something like LockException) so that all those problems could be catch using a commons exception superclass and handled, if needed. what do you think?

Reply:
done here b2a0f34178f6fb210f8722775413d7fd82975ec4